.github/workflows/publish.yml

```diff
@@ -1,11 +1,11 @@
 name: Publish
 
 on:
-  workflow_run:
-    workflows: [Python]
-    types: [completed]
+  push:
     branches:
       - "v[0-9]+.[0-9]+.x"
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
```

Comment:
Ok, leaving this conv open for us to check if this is the case.

Reply:
I tested adding a tag. it runs it once for push only. There is no link between a publish and test though so it will be up to us to check stuff before releasing. If it creates a problem we can look into making this dependent on pytest output.